kie-archetypes/kie-service-spring-boot-archetype/README.md

```diff
@@ -167,7 +167,7 @@ The default value of this property is the archetype groupId.
 
```

Comment:
I would personally include `-DswaggerEnabled=true` in that initial section of the README, which includes the example mvn commands (one multi line, second is the one liner).

Swagger is universally used to talk to kie-server in dev environment, and in my opinion it would be a sensible default to have it enabled.

Reply:
Done. So by the default, you will always get swagger enabled now